.vib/grafana-operator/cypress/cypress/integration/grafana_operator_spec.js

```diff
@@ -2,6 +2,13 @@
 import { random } from '../support/utils';
 
 it('allows uploading a dashboard as JSON file', () => {
+  cy.on('uncaught:exception', (err, runnable) => {
```

Comment:
As a best practice, this logic is added to the `commands.js` file so it affects every step of the test suite. This way we can also maintain the test code cleaner. Could you please move it there? You can find some examples of it in other `commands.js` files.